tasks/main-blivet.yml

```diff
@@ -145,10 +145,14 @@
 - name: manage /etc/crypttab to account for changes we just made
   lineinfile:
     path: /etc/crypttab
+    # The line option is used when `state: present`
     line: "{{ entry.name }} {{ entry.backing_device }} {{ entry.password }}"
+    # The regexp option is used when `state: absent` to remove entries
+    # regardless of password
+    regexp: ^{{ entry.name }} {{ entry.backing_device }}
     state: "{{ entry.state }}"
     create: true
-    mode: "{{ __storage_crypttab.stat.mode | d('0600') }}"
+    mode: "{{ __storage_crypttab.stat.mode | d('0622') }}"
```

Comment:
this is an unusual mode - `-rw--w--w-` - the file has to be writable by all but not readable?

Reply:
Sorry my bad, it should be `-rw-r--r--`, changed to 0644 now.

Reply:
does the file have to be world-readable?  It has a password in it?

Reply:
On master, the role does `0600`. I mistakenly used `0644` because that's what lineinfile does by default. The file indeed stores an unencrypted password in it, so `0600` is the way to go. Sorry for the confusion and thank you for the thorough review!